ihmc-footstep-planning/src/main/java/us/ihmc/footstepPlanning/SnappingTerrainExtractor.java

```diff
@@ -26,7 +24,7 @@
 
 public class SnappingTerrainExtractor
 {
-   private static final boolean PRINT_TIMING_FOR_KERNELS = false;
+   private static final boolean PRINT_TIMING_FOR_KERNELS = true;
```

Comment:
Probably should keep this false by default
```suggestion
   private static final boolean PRINT_TIMING_FOR_KERNELS = false;
```